src/status_im/contexts/wallet/wallet_connect/core.cljs

```diff
@@ -23,9 +32,14 @@
   [event]
   (get-in event [:params :request :params]))
 
+(defn get-db-current-request-event
```

Comment:
Do we really need this function. In Clojure land, it's looked down upon to have DSLs for operations where the standard language API can be used. 

What's wrong with `(get-in db [:wallet/connect ...])` ? Why extract this to a function?

Reply:
Doesn't matter in the context of this PR — the function `get-db-current-request-params` was there before, have no idea who wrote it but I just moved one part of the logic to another function for the sake of one particular problem I was solving here.